src/modules/mavlink/streams/BATTERY_INFO.hpp

```diff
@@ -67,36 +67,49 @@ class MavlinkStreamSmartBatteryInfo : public MavlinkStream
 
 			if (battery_sub.update(&battery_status)) {
 				if (battery_status.serial_number == 0) {
-					// This is not smart battery
+					// Required to emit
 					continue;
 				}
 
-				mavlink_smart_battery_info_t msg{};
+				mavlink_battery_info_t msg{};
 
 				msg.id = battery_status.id - 1;
-				msg.capacity_full_specification = battery_status.capacity;
-				msg.capacity_full = (int32_t)((float)(battery_status.state_of_health * battery_status.capacity) / 100.f);
+				msg.design_capacity = (float)(battery_status.capacity * 1000);
+				msg.full_charge_capacity = (float)(battery_status.state_of_health * battery_status.capacity * 1000.f) / 100.f;
 				msg.cycle_count = battery_status.cycle_count;
 
 				if (battery_status.manufacture_date) {
 					uint16_t day = battery_status.manufacture_date % 32;
 					uint16_t month = (battery_status.manufacture_date >> 5) % 16;
-					uint16_t year = (80 + (battery_status.manufacture_date >> 9)) % 100;
+					uint16_t year = (80 + (battery_status.manufacture_date >> 9));
+					uint16_t year2dig = year % 100;
 
+					//Formatted as 'ddmmyyyy' (maxed 9 chars)
+					snprintf(msg.manufacture_date, sizeof(msg.manufacture_date), "%d%d%d", day, month, year);
 					//Formatted as 'dd/mm/yy-123456' (maxed 15 + 1 chars)
-					snprintf(msg.serial_number, sizeof(msg.serial_number), "%d/%d/%d-%d", day, month, year, battery_status.serial_number);
+					snprintf(msg.serial_number, sizeof(msg.serial_number), "%d/%d/%d-%d", day, month, year2dig, battery_status.serial_number);
 
 				} else {
+					
 					snprintf(msg.serial_number, sizeof(msg.serial_number), "%d", battery_status.serial_number);
 				}
 
-				//msg.device_name = ??
-				msg.weight = -1;
-				msg.discharge_minimum_voltage = -1;
-				msg.charging_minimum_voltage = -1;
-				msg.resting_minimum_voltage = -1;
-
-				mavlink_msg_smart_battery_info_send_struct(_mavlink->get_channel(), &msg);
+                // Not supported by PX4 (not in battery_status uorb topic)
+				/*
+				msg.name = 0; // char[50]
+				msg.weight = 0;
+				msg.discharge_minimum_voltage = 0;
+				msg.charging_minimum_voltage = 0;
+				msg.resting_minimum_voltage = 0;
+				msg.charging_maximum_voltage = 0;
+				msg.charging_maximum_current = 0;
+				msg.discharge_maximum_current = 0;
+				msg.discharge_maximum_burst_current = 0;
+				msg.cells_in_series = 0;
+				msg.nominal_voltage = 0;
+				*/
```

Comment:
My understanding is that if the default is 0 you don't need to set it. I've kept here to show what we haven't implemented yet in the MAVLink message.